.github/workflows/coverage.yml

```diff
@@ -100,48 +107,63 @@ jobs:
         name: test_coverage
         path: test-coverage/
 
-  publish_coverage:
-    name: Publish coverage
+  check_coverage:
+    name: Check coverage
     needs: generate_coverage
-    concurrency: gh_pages
-    if: github.event_name == 'push' && needs.changes.outputs.tket == 'true'
+    if: (github.event_name == 'push' && needs.changes.outputs.tket == 'true') || github.event_name == 'schedule'
```

Comment:
We want to run this job on `pull_request`, not `push`. (Coverage reduction should be caught at PR stage.)

On the other hand, the "publish" job should run on `push`. So the dependencies need tweaking a bit. (I think `publish_coverage` needs `generate_coverage` but not `check_coverage`?)

Reply:
Addressed :)